### PR TITLE
Feat/lint fixes

### PR DESCRIPTION
**Pull request by jonty007**

### CODE REVIEW

File: packages/components/src/SmartObject.tsx [hunk 1]
```diff
@@ -48,112 +48,6 @@ const SmartObjectValues = ({ smartObject }: any) => {
   )
 }
 
-const revToId = (rev: string) => rev?.split(":")[0]
```
Comment (ClemensLey) [inline packages/components/src/SmartObject.tsx:51]: We should add the functions revToId and MetaData back in but comment them out for now

File: packages/components/src/Auth.tsx [hunk 2]
```diff
@@ -339,8 +339,6 @@ function LoginButton({ mnemonic, chain, network, path, url }: any) {
       if (isLoggedIn()) throw new Error("A user is already logged in, please log out first.")
```
Comment (ClemensLey) [inline packages/components/src/Auth.tsx:339]: We can remove the try catch block and just call showSnackBar directly instead

File: packages/wallet/src/components/Utils/Drawer.tsx [hunk 3]
```diff
@@ -53,14 +52,6 @@ export const CustomDrawer = ({ id, computer }: any) => {
     initFlowbite()
   }, [])
 
-  const refreshBalance = useCallback(async () => {
```
Comment (ClemensLey) [inline packages/wallet/src/components/Utils/Drawer.tsx:56]: I'm not sure if we might need this block

Comment (jonty007) [reply packages/wallet/src/components/Utils/Drawer.tsx:56]: have added it and kept it as a comment

**Pull request merged**